app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :comments, foreign_key: 'author_id'
+
```

Comment:
- [ ] as you can see in the below image from the ERD diagram there should be a has_many relationship for the likes as well in the user class.
![afbeelding](https://github.com/Zeph180/blog-it/assets/39240820/47a2eac2-ee1d-4c05-9ca5-e0726b90a592)